### DISCUSSION TITLE
ecal_sys_gui remote control

### DISCUSSION THREAD

**Discussion by ENAC-Robotique** [eclipse-ecal/ecal#1855] [Q&A]
Hi,
I am currently using systemd (linux services) to manage my programs on a headless robot, and I would like to switch to ecal_sys, mainly because of the nice GUI.
ecal_sys is better for me thank to the remote control feature, but it seems that ecal_sys_gui can't remote control an ecal_sys instance.
Am I missing something here ?

**Comment by rex-schilasky** [comment]
eCALSys is able to remote control applications as well. It is using ecal sys client applications on remote hosts. You need to start these "demons". They are addressed from eCALSys GUI via RPC to start or shutdown local processes then.
